providers/fips/fipsprov.c

```diff
@@ -85,6 +89,31 @@ static const OSSL_PARAM fips_param_types[] = {
     OSSL_PARAM_END
 };
 
+/*
+ * Parameters to retrieve from the core provider - required for self testing.
+ * NOTE: inside core_get_params() these will be loaded from config items
+ * stored inside prov->parameters.
```

Comment:
except `OSSL_PROV_PARAM_MODULE_FILENAME` which is not loaded from config

Reply:
Updated the comment